### ISSUE TITLE
Enhance NFSv4 support (in particular for more security than NFSv3)

### ISSUE THREAD

**Issue by jsmeix** [rear/rear#754]
In a DMZ we can not use the normal backup solution
and we are stuck to netfs and NFS.
Currently we use NFSv3 which has totally no security.
In NFSv4 we have some more options.

AFAIK rear does not yet support NFSv4 because
rear does not deal with the extra daemons
that it needs (idmap ...).

Currently NFSv4 works only as much as
the plain kernel module supports.

Adding the daemons should be not difficult. See
usr/share/rear/verify/NETFS/default/05_start_required_daemons.sh
what already happens for NFSv3.

**Comment by gdha** [comment]
@jsmeix I think this issue deserves to be completed for rear-1.19, don't you think so? More and more NFSv4 is being used. If I can I'll give it a try in the near future.

**Comment by jsmeix** [comment]
Right now I was wondering about my initial comment https://github.com/rear/rear/issues/754#issue-126609225
because I know that this is not my own wording.

I found out wherefrom it came:

I did only some copy and paste from the
"ReaR and NFS4" mail thread in Jan. 2016
on the Relax-and-Recover users list, see
http://pikachu.3ti.be/pipermail/rear-users/2016-January/thread.html
in particular see
http://pikachu.3ti.be/pipermail/rear-users/2016-January/003282.html

Personally I know nothing at all about NFSv4 options
and currently I do not have any customer request
for it so that from my current point of view
there is no current demand for special NFSv4 options.

In particular because there is not any kind of
futher communication from Michael Brookhuis here
I set the milestome to any unspecific future rear version.

@gdha 
nevertheless I would of course appreciate it
if you like to implement some first steps
towards specific NFSv4 support so that
users who need specific NFSv4 support
could find some entry points where they can
further enhance it according to their needs.
